projects/chaos/inc/charger.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+// Charger module for interacting with the charger board over CAN.
+//
+// Requires CAN to be enabled.
+
+#include "exported_enums.h"
+#include "status.h"
+
+// Configure the charger and set up CAN handlers.
+StatusCode charger_init(void);
+
+// Sets the charger state.
+// - If enabled it will only be active when the charger is connected to the charger board.
+// - If disabled it will always be off regardless of whether a charger is connected.
+StatusCode charger_set_state(EEChargerSetRelayState state);
```

Comment:
How does chaos know when to set the state? For example, if we're in charge mode before the charger is plugged in.

Reply:
I'm planning to create an Event to trigger this when entering/exiting the charging state using the sequencer. I was planning to leave that to a separate PR.